changelog.d/2214.contrib.md

```diff
@@ -0,0 +1 @@
+`pip-tools` now tests on and officially supports `pip` version 25.2 .
```

Comment:
NB: I think a byline may be missing here?  (noticed during preparation of #2217)

```suggestion
`pip-tools` now tests on and officially supports `pip` version 25.2 .
 -- by {user}`sirosen`
```

Reply:
I don't mind always including the byline if it helps us keep the style consistent for contributor PRs.

I have a couple of projects in which I give "`Thanks @user`" for 3rd party contributions and not for maintainer-driven changes, which is probably the habit that led to not having one here.